es6-shim.js

```diff
@@ -1387,7 +1387,8 @@
     var isBadHex = isBadHexRegex.test.bind(isBadHexRegex);
     var NumberShim = (function () {
       // this is wrapped in an IIFE because of IE 6-8's wacky scoping issues with named function expressions.
-      var NumberShim = function Number(value) {
```

Comment:
this would be a breaking change; `Number.length` must be `1`.

Reply:
I missed that requirement! Thanks.
I will need to think about this some more then, just by having the formal parameter, even if only arguments is used, IE11 is much slower than only having formal parameters or only having arguments.

Reply:
Unfortunately function lengths are not configurable in IE 11, so the only way to get a length of 1 is to have a formal parameter.